__tests__/endpoint.test.js

```diff
@@ -47,4 +47,41 @@ describe('/api', () => {
             };
           });
       });
+})
+describe('/api/articles/:article_id', () => {
+    test("GET:200 sends an array of topics to the client", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({body}) => {
+            const {article} = body
+            expect(article).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            article_img_url:
+                "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            })
```

Comment:
I'd consider changing this test to allow for later extension of the endpoint, the test should use a flexible matcher eg **expect.objectContaining** or **.toMatchObject.** 👀